packages/cli/src/commands/serve.ts

```diff
@@ -13,16 +15,45 @@ class PlatformAppDevelopmentServer {
 
     async serve(): Promise<void> {
         try {
-            const viteServer = await createServer({
+            const settingsSchema = await esbuild.context({
+                entryPoints: ['./src/index.ts'],
```

Comment:
It is set on line 67
```suggestion
                entryPoints: [entryFilePath],
```